static/js/blogSingle.js

```diff
@@ -0,0 +1,102 @@
+// share on twitter
+
+function shareOnTwitter() {
+  const twitterWindow = window.open(
+    'https://twitter.com/share?url=' + document.URL,
+    'twitter-popup',
+    'height=350,width=600'
+  );
+  if (twitterWindow.focus) {
+    twitterWindow.focus();
+  }
+  return false;
+}
+
+// share on tiwtter with title , url
+function shareOnTwitterWithTitle() {
+  const title = document.querySelector('[data-blog-title]').innerText;
+  const twitterWindow = window.open(
+    'https://twitter.com/intent/tweet?text=' +
+      title +
+      '&url=' +
+      document.URL +
+      '&via=rotationalio',
+    'twitter-popup',
+    'height=350,width=600'
+  );
+  if (twitterWindow.focus) {
+    twitterWindow.focus();
+  }
+  return false;
+}
+
+//share on linkedin
+function shareOnLinkedIn() {
```

Comment:
Another follow on story is required here because the LinkedIn window doesn't include a link to blog posts when the share button is clicked.